apps/shell/tash_command.c

```diff
@@ -191,6 +193,15 @@ static int tash_help(int argc, char **args)
 	return 0;
 }
 
+/** @brief Clear the terminal screen and move cursor to top
+ *  @ingroup tash
+ */
+static int tash_clear(int argc, char **args)
+{
+	printf("\x1b[1;1H\x1b[2J\n");
```

Comment:
As I Remember, this is kind of ANSI escape code, right?
If you have some time later, define these of values to somewhere(header file will be enough).